proto/buf/alpha/registry/v1alpha1/webhook.proto

```diff
@@ -42,24 +44,26 @@ message CreateWebhookRequest {
 
 // WebhookEvent contains the currently supported webhook event types.
 enum WebhookEvent {
-  // WEBHOOK_EVENT_UNSPECIFIED will result in a noop for registering webhooks.
+  // WEBHOOK_EVENT_UNSPECIFIED is a safe noop default for webhook events
+  // subscription. It will trigger an error if trying to register a webhook with
+  // this event.
   WEBHOOK_EVENT_UNSPECIFIED = 0;
-  // WEBHOOK_EVENT_REPOSITORY_PUSH is emitted whenever a successful buf push
-  // is completed for a specific repository.
+  // WEBHOOK_EVENT_REPOSITORY_PUSH is emitted whenever a successful buf push is
+  // completed for a specific repository.
   WEBHOOK_EVENT_REPOSITORY_PUSH = 1;
 }
 
-// CreateWebhookResponse is the proto repsonse representation
+// CreateWebhookResponse is the proto response representation
```

Comment:
Unrelated to this pr, but in general we haven't documented the `Foo{Request,Response}` types as they're tightly coupled to one service, for which the method should really have the comment on what it does.

Reply:
Done